src/MHI-AC-Ctrl.h

```diff
@@ -23,6 +23,7 @@
 #define MODE_COOL = "Cool";
 #define MODE_FAN = "Fan";
 #define MODE_HEAT = "Heat";
+#define MODE_ERROR = "Stop";
```

Comment:
Is it better to rename it to MODE_STOP?

Reply:
I wasn't really sure what the "Stop" mode was used for, there's a comment in the ino file on line 106 that it is "for error operating data". So I named it like that.
Might be better if this was the following, but I didn't want to break any existing implementations.
`MODE_ERROR = "Error"`